controllers/articles.controllers.js

```diff
@@ -11,18 +11,34 @@ function getArticlesById (req,res,next){
 }
 
 
-//task5 & task 11
+
+//task5 & task11
+
+
 
 function getArticles(req,res,next){
     
 
-    return selectArticle()
+
+    const validQueries = ['sort_by', 'order'];
+    const receivedQueries = Object.keys(req.query);
+
+    // Check for unsupported query parameters
+    const invalidQueries = receivedQueries.filter(query => !validQueries.includes(query));
+
+    if (invalidQueries.length > 0) {
+        return res.status(400).send({ msg: 'Bad request: invalid query parameter' });
+    }
+    
```

Comment:
It's very considerate that you've covered the invalid query param, but there will be topic as a filter later, so make sure to add topic into the validQueries array later or remove the validQueries array. If there's invalid query, it'll be ignored instead of throwing an error message, which is also OK